### ISSUE TITLE
aiming keep working in emote menu

### ISSUE THREAD

**Issue by PromHeda** [cmooref17/Lethal-Company-TooManyEmotes#40]
hi everyone, its probably something i can fix from config file but i couldn't manage to do it. After updating game to v49 my aim start working when i open emote menu so when i move my mouse to select an emote my characters aim still goes with my mouse which is a bummer when i encounter with coilhead.

Tried reinstalling mod
tried changing menu key
tried reinstalling game
tried different config settings

But none of them worked.

I would appreciate if you can help me with it.

**Comment by cmooref17** [comment]
So you're saying when you open up the emote menu, moving your mouse rotates your character? Like he'll start looking up/down/left/right? That doesn't seem right. Let me look into this when I get home.

**Comment by PromHeda** [comment]
> So you're saying when you open up the emote menu, moving your mouse rotates your character? Like he'll start looking up/down/left/right? That doesn't seem right. Let me look into this when I get home.

Yep thats exactly what happens but my other friends doesnt have this issue also after updating the mod yesterday they stopped working none of the emotes work, when i try to do one it just gets stuck. i tried uninstalling and installing mod back but didn't work

**Comment by PromHeda** [comment]
> > So you're saying when you open up the emote menu, moving your mouse rotates your character? Like he'll start looking up/down/left/right? That doesn't seem right. Let me look into this when I get home.
> 
> Yep thats exactly what happens but my other friends doesnt have this issue also after updating the mod yesterday they stopped working none of the emotes work, when i try to do one it just gets stuck. i tried uninstalling and installing mod back but didn't work

so i found the issue its because of the minimap mod after uninstalling it everything seems working again

**Comment by cmooref17** [comment]
Oh, interesting. I haven't heard about that compat issue yet. Thanks for letting me know!
